### PR TITLE
[IMPROVEMENT] Refactor ProtectYourWallet section in the security section

### PR DESCRIPTION
**Pull request by gantunesr**
**Description**

Refactor ProtectYourWallet section in the security section to TS, functional component, and move the code to their on file.

**Screenshots/Recordings**

~_If applicable, add screenshots and/or recordings to visualize the before and after of your change_~

**Issue**

Part of https://github.com/MetaMask/mobile-planning/issues/591

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/Views/Settings/SecuritySettings/Sections/ProtectYourWallet/ProtectYourWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        source: 'Settings',
+      });
+    });
+  };
+
+  const onBack = (): void => navigation.goBack();
+
+  return (
+    <View style={[styles.setting, styles.firstSetting]}>
+      <Text style={[styles.title, styles.bump]}>
```
Comment (blackdevelopa) [inline app/components/Views/Settings/SecuritySettings/Sections/ProtectYourWallet/ProtectYourWallet.tsx:63]: Just a small nitpick and completely at your discretion. Can we use the `Text` element from the component-library?

**Review by blackdevelopa — COMMENTED** [review]
LGTM. 🙌🏿

File: app/components/Views/Settings/SecuritySettings/Sections/ProtectYourWallet/ProtectYourWallet.tsx (hunk 1 above)
Comment (gantunesr) [reply app/components/Views/Settings/SecuritySettings/Sections/ProtectYourWallet/ProtectYourWallet.tsx]: Good call.

I change the `Button` and `Text` component for the DS ones

Comment (gantunesr) [reply app/components/Views/Settings/SecuritySettings/Sections/ProtectYourWallet/ProtectYourWallet.tsx]: Button: [d1e6205](https://github.com/MetaMask/metamask-mobile/pull/5600/commits/d1e6205d4a15454a308a527905a9335c9f3c59fd)

Text: [46fca0d](https://github.com/MetaMask/metamask-mobile/pull/5600/commits/46fca0de1ecf4a55889f45a07d6e347abdcf9a7d)

Comment (blackdevelopa) [reply app/components/Views/Settings/SecuritySettings/Sections/ProtectYourWallet/ProtectYourWallet.tsx]: Looks good to me.

**Review by blackdevelopa — APPROVED** [review]

**Pull request merged**